### ISSUE TITLE
How do i filter for only dead animals?

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#5204]
**How do i filter for only dead animals?**

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/63b92a69e54c4a51803a55fb1dea7123:97a920d2e20d5343acad5c0b9e7d505a12f8c4ac883a9524346aba53a3740f754ac9d101a0ec149b66e090a928a473e94db3889ea9756888072721d0632b35e6ea5fc23be2a76f6cb57f806f1bc2b91e09d777f7ef4bfde25c10fa2bbd4db733) - [Send email](https://www.gbif.org/api/feedback/user/mailto/63b92a69e54c4a51803a55fb1dea7123:97a920d2e20d5343acad5c0b9e7d505a12f8c4ac883a9524346aba53a3740f754ac9d101a0ec149b66e090a928a473e94db3889ea9756888072721d0632b35e6ea5fc23be2a76f6cb57f806f1bc2b91e09d777f7ef4bfde25c10fa2bbd4db733)
System: Chrome 121.0.0 / Windows 10.0.0
Referer: https://www.gbif.org/
Window size: width 1707 - height 898
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2024-02-22T15:40:40.497Z',to:'2024-02-22T15:46:40.497Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2024-02-22T15:40:40.497Z',to:'2024-02-22T15:46:40.497Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by ManonGros** [comment]
We currently don’t have an easy way to filter for observations or collection of dead animals.

As you might know, GBIF is a data aggregator. The data avialable on our platform comes from many different sources: Museums, citizen science intiatives, research teams, etc. Data providers use specific data standards to share data on GBIF. Until recently, there was no field dedicated to specifying if a specimen was found dead.

This means that the information was shared in various places and isn’t always easy to find. For example, searching GBIF occurrences for the word “dead” will give you all kinds of results (see the outcome of the search [here](https://www.gbif.org/occurrence/search?q=dead&taxon_key=1&occurrence_status=present)). Some occurrences like this one https://www.gbif.org/occurrence/2907213645 will have values in the occurrenceRemark field mentioning that the specimen “Found dead on road”. Other occurrence might mention the word dead to refer to other species than the one observed like here: https://www.gbif.org/occurrence/3007121726.
In other words, using this type of key words isn’t very reliable.

You could also try looking for specific datasets which might contain occurrences od dead animals. For example, there are a number of roadkill datasets on GBIF: https://www.gbif.org/dataset/search?q=roadkill

Note that there is now a field called vitality (https://dwc.tdwg.org/terms/#dwc:vitality) which would allow data providers to specify if an organism was alive or dead at the time of the observation. However, this field is very new in the standard. Most data providers wouldn’t have filled the information yet and it isn’t yet a filter on our system.

> replied by email
